grudge/metadata.py

```diff
@@ -0,0 +1,44 @@
+from pytools.tag import UniqueTag, tag_dataclass
+from .dof_desc import DOFDesc
+
+
+@tag_dataclass
+class DiscretizationElementAxisTag(UniqueTag):
+    """
+    Tagged to an array's axis travesing elements in a discretization.
```

Comment:
- typo: traversing
- traversing -> "ranging across"? "enumerating"? "representing element indices"?
- "in a discretization" -> "in the discretization identified by :attr:`dd`"